komodoenv/creator.py

```diff
@@ -158,9 +115,6 @@ def create(self):
         ) as f:
             print("\n".join(python_paths), file=f)
 
-        # Create shims
-        self.shim_pythons()
```

Comment:
Was this just for the sake of py27 compatibility?

Reply:
This was for RHEL7 compatibility due to our misuse of `LD_LIBRARY_PATH`.